packages/base/src/Overlay/Overlay.tsx

```diff
@@ -50,6 +52,7 @@ export const Overlay: RneFunctionComponent<OverlayProps> = ({
   ...rest
 }) => (
   <ModalComponent
+    animationType={animationType}
```

Comment:
I don't think this will make any effect, this prop is passing via `...rest` anyways.